java/bin/start.sh

```diff
@@ -16,6 +16,31 @@ set -e
 # limitations under the License.
 
 #Initialize functions and Constants
+echo "Script Started Execution"
+
+# Auxiliary Function to check the exit status passed as an argument
+# This function is also responsible for printing the error message or success message based on the exit status
+check_status()
```

Comment:
Thankyou for making the changes.
it will be great if we move this function in dataproc_template_functions.sh

Reply:
Done.